Automattic/wp-calypso
# Fix unlocalized links everywhere

#### Changes proposed in this Pull Request

The previous PR, #63116, fixed specific unlocalized URLs in the Domain Management System. This PR fixes issues that were excluded from  #63116 .


#### Testing Instructions

1. Go to https://wordpress.com/domains/manage/ and select a site with a domain.
2. Click on the 3 dots to see the domain settings.
3. Choose from the following options:
- DNS records->Manage
- Transfer
4. Verify that the support URLs on the page have been localized. 


Relates to 381-gh-Automattic/i18n-issues
